### DISCUSSION TITLE
Split payment functionality

### DISCUSSION THREAD

**Discussion by magdafiktus** [medusajs/medusa#4430] [Feature Requests]
how to deal with inventory? medusa can only process one payment method per cart

**Comment by srindom** [comment]
We are going to need some more inputs on what you mean here. For example:

- What problem do you have that you would like Medusa to solve?
- How are you currently solving this problem?
- What would an ideal solution do?

**Reply by magdafiktus** [reply]
Hey, of course! So we were trying to think what kind of flow we could have to allow for a split payment on the POS, so the steps could be

1. split payment is chosen on the payment screen
2. methods - cash and credit card
3. first process cash
4. continue and create the second payment with terminal for credit card
5. show single confirmation screen and display them both
6. somehow send data to medusa to show split payment
how to deal with the inventory? medusa can only process one payment method per cart

We are not solving it now, it is unavailable.

Ideally we could process two payments in one transaction.

Another solution describes it as "If a customer can't pay for their whole order, then the ... app lets you to accept partial payments in store. You can retrieve the order later, and then collect the remaining balance from the customer using a payment method of their choice."

**Reply by lexvz14** [reply]
Hi there! How did you solve this challenge in the end? I'm investigating how to go about this too. If a cart can only have one payment related to it, that will be a challenge indeed.
